integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -71,7 +71,7 @@ master:
     embedded: 19200
     rpc: 19998
     web: 19999
-  hostPID: true
+  hostPID: false
```

Comment:
our default pod security policy is strict enough to disallow `hostPID`?

Reply:
please add a comment here that a value of true requires escalated privileges (which is kind of obvious but still)

Reply:
I copied the PSP from best practices, which disabled hostPID.

Reply:
Done